### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #18 [December] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#10039] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, December 14 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/9955
- Next: https://github.com/celo-org/celo-monorepo/discussions/10079
- Recording: [Google Drive](https://drive.google.com/file/d/1DTtXAn_aAzaWmeieyrCBJrSqo2dtL-gu/view?usp=sharing)

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [x] RECORD meeting 🎥 
- [x] Share recording below and on forum
- [x] Create next meeting (link to next meeting above)

**Comment by Michaelcelo** [comment]
**Agenda Item to add** - [Fiatconnect API](https://github.com/fiatconnect/specification) and Connect the World Update
Silas and I will provide an update on the CICO Partners going live soon, benefits for end-users, and other adjacent initiatives!

**Reply by Michaelcelo** [reply]
We would like to have ~10 minutes carved out in the agenda to present this

**Reply by arthurgousset** [reply]
Meeting notes: 
- Successful e2e test with AlphaFortress (will launch in Nigeria). This will bring support to new countries for cash out
- Bitmama very close to passing e2e tests.
- 2 providers 🥳 
- There are centralised CICO aggregators (e.g. onramper)

**Comment by jeanregisser** [comment]
**Agenda item to add:** WalletConnect V2 support in Valora
Quick demo.

**Reply by arthurgousset** [reply]
Meeting notes: 
- Valora implemented WCv2 ~1 year ago (was unstable then)
- WCv2 is not backwards compatible
- Very detailed transition plan: (1) wallets support WCv2 first, (2) dapps support WCv2
- Valora has WCv2 support in `v1.47` 🥳 
- WCv2 does not address deep linking issues
- Deadline: March 1, 2023 for dapps to upgrade to WCv2

**Reply by jeanregisser** [reply]
Official migration plan: https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e

**Comment by arthurgousset** [comment]
**Agenda item to add**: ASv2 demo payment between Kaala and ImpactMarket and FAQ on ASv2 (if any).
Also chat about wallets' cross-chain ambitions.

**Reply by arthurgousset** [reply]
Meeting notes: 
- Valora saw great success with centralised phone number verification (60% -> 93% success rate)
- Valora also saw increasing activation rates (person onboards with funds in account) from 4-8% -> 20-50%.
- Node wallet integrating by late Jan/early Feb

**Comment by arthurgousset** [comment]
Here is the [recording](https://drive.google.com/file/d/1DTtXAn_aAzaWmeieyrCBJrSqo2dtL-gu/view?usp=sharing) for future reference.

**Comment by Ashimar** [comment]
谢谢！已收到！
